Automattic/jetpack
# Subscribe: Transparent button border fix

Fixes https://github.com/Automattic/jetpack/issues/36244

## Proposed changes:

It fixes the issue with missing Subscribe button border by removing the `border-color: transparent;` style.

Didn't notice any regression after removing it. It was added here: https://github.com/Automattic/jetpack/pull/15107

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* activate Hello Elementor theme
* insert Subscribe block into the post
* open the post and make sure the button has a border visible
